src/aioice/turn.py

```diff
@@ -39,6 +39,14 @@ def make_integrity_key(username: str, realm: str, password: str) -> bytes:
     return hashlib.md5(":".join([username, realm, password]).encode("utf8")).digest()
 
 
+def calc_padding(value):
```

Comment:
How about a nice docstring referencing the RFC? :)

Reply:
Fixed.